src/data/ingredientToIngredientAliases.ts

```diff
@@ -155,6 +155,21 @@ const ingredientToIngredientAliases: IngredientToIngredientAliases = {
   "Boiling Water": [],
   "Cinnamon": [],
   "Pumpkin Syrup": [],
+  "Brandy": [],
+  "Apple Cider": [],
+  "Pear Brandy": [],
+  "Hard Apple Cider": [],
+  "Dry Alsatian Riesling": [],
+  "Star Anise Pod": [],
+  "Ginger": [],
+  "Whole Cloves": [],
```

Comment:
When I googled this ingredient, I learned that if you just search "clove" these same things come up. The only clove I'd heard of until then was a garlic clove. Anyway - should this have a "clove" alias?